components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -4336,31 +4336,49 @@ func TestCreateProject(t *testing.T) {
 	ctx := context.Background()
 
 	cases := map[string]func(*testing.T){
-		"successfully creates custom project": func(t *testing.T) {
+		"successfully creates chef-managed project": func(t *testing.T) {
+			project := storage.Project{
+				ID:     "my-id-1",
+				Name:   "name1",
+				Type:   storage.ChefManaged,
+				Status: storage.NoRules.String(),
+			}
+			resp, err := store.CreateProject(ctx, &project, false)
+			require.NoError(t, err)
+			require.Equal(t, &project, resp)
+
+			assertProjectsMatch(t, db, project)
+		},
+		"successfully creates custom project without supporting policies": func(t *testing.T) {
 			project := storage.Project{
 				ID:     "my-id-1",
 				Name:   "name1",
 				Type:   storage.Custom,
 				Status: storage.NoRules.String(),
 			}
-			resp, err := store.CreateProject(ctx, &project)
+			resp, err := store.CreateProject(ctx, &project, false)
 			require.NoError(t, err)
 			require.Equal(t, &project, resp)
 
 			assertProjectsMatch(t, db, project)
+			assertEmpty(t, db.QueryRow(`SELECT count(*) FROM iam_policies`))
 		},
-		"successfully creates chef-managed project": func(t *testing.T) {
+		"successfully creates custom project with supporting policies": func(t *testing.T) {
 			project := storage.Project{
 				ID:     "my-id-1",
 				Name:   "name1",
-				Type:   storage.ChefManaged,
+				Type:   storage.Custom,
 				Status: storage.NoRules.String(),
 			}
-			resp, err := store.CreateProject(ctx, &project)
+			assertEmpty(t, db.QueryRow(`SELECT count(*) FROM iam_policies`))
+			resp, err := store.CreateProject(ctx, &project, true)
 			require.NoError(t, err)
 			require.Equal(t, &project, resp)
 
 			assertProjectsMatch(t, db, project)
+			assertCount(t, 3, db.QueryRow(
+				`SELECT count(*) FROM iam_policies
+				 WHERE id IN ('my-id-1-project-owners','my-id-1-project-viewers','my-id-1-project-editors')`))
 		},
```

Comment:
This is the new test to confirm the policies are created.
All other changes in this file are overhead.